src/app/visit/status.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <string>
+
+enum class Status
```

Comment:
Despite adding Unassigned status think of renaming it to be more descriptful.
Status might be status of visit, status online etc. 
As it's directly connected to visit status, VisitStatus seems fine, but you might find something better here.

Reply:
Done